src/pages/community/events.js

```diff
@@ -28,14 +28,14 @@ const PAGE_DESCRIPTION =
 
 export default () => {
     const [events, error, isLoading] = useEventData();
-    const { title, siteUtl } = useSiteMetadata();
+    const { title } = useSiteMetadata();
 
     return (
         <Layout>
             <PageHelmet
-                metaTitle={`Events - ${title}`}
-                canonicalUrl={`${siteUtl}/community/events/`}
-                metaDescription={PAGE_DESCRIPTION}
+                title={`Events - ${title}`}
+                pagePath="/community/events"
```

Comment:
For `pagePath` Gatsby gives this component `props`, one of which is `uri` and another is `path` which might be useful here so we don't have to hardcode. If the pagePath is the `uri` we should use that. What do you think?

Reply:
Thanks! Nice suggestion I will add.